discovery-infra/start_discovery.py

```diff
@@ -311,6 +314,9 @@ def main():
         type=str,
         default="redhat.com",
     )
+    parser.add_argument(
+        "-mD", "--managed-dns-domains", help="Managed DNS domains", type=str, default=""
```

Comment:
cam u maybe write more descriptive help here. Not sure that I undestand what should be provided. Even give an example

Reply:
sure, added explanation and example, also updated the readme in: https://github.com/openshift/assisted-test-infra/pull/16

Reply:
added DNS validation in: https://github.com/openshift/assisted-test-infra/pull/21